example/demo/main.zig

```diff
@@ -1,15 +1,14 @@
 const std = @import("std");
-const template = @import("demo-ztt");
-
+const renderer = @import("renderer");
```

Comment:
I would call it `template`, as that's what it is. We can execute the code in it (`render`) to fill out the template.